custom_components/tahoma/cover.py

```diff
@@ -56,12 +56,13 @@ async def async_setup_entry(hass, entry, async_add_entities):
 
     data = hass.data[DOMAIN][entry.entry_id]
 
-    entities = []
     controller = data.get("controller")
 
-    for device in data.get("devices"):
-        if TAHOMA_TYPES[device.uiclass] == "cover":
-            entities.append(TahomaCover(device, controller))
+    entities = [
```

Comment:
Would be good to port this change to all platforms as well, in a future PR.

Reply:
I will.